src/NuGet.Core/NuGet.Build.Tasks.Pack/PackTaskLogic.cs

```diff
@@ -447,6 +447,19 @@ private IEnumerable<OutputLibFile> InitLibFiles(IMSBuildItem[] libFiles)
                             string.Format(CultureInfo.CurrentCulture, Strings.InvalidPlatformVersion, targetFramework)
                         );
                     }
+                    var isNet5EraTfm = fw.Version.Major >= 5 &&
+                        StringComparer.OrdinalIgnoreCase.Equals(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, fw.Framework);
+                    var isNetStandard = StringComparer.OrdinalIgnoreCase.Equals(FrameworkConstants.FrameworkIdentifiers.NetStandard, fw.Framework);
+                    if (isNet5EraTfm || isNetStandard)
```

Comment:
Why are we checking the netstandard versions?

Reply:
https://github.com/nuget/home/issues/9215#issuecomment-608658914

Reply:
There'll be no new versions of netstandard. 
The true risk here is what happens when we hit `net10.0`. The risk is not there with netstandard.

Keep in mind that there are existing packages/projects with netstandard. I see no reason to raise an extra warning for something that won't really affect them.